checker/jtreg/stubs/issue2059/AnnoNotFound.out

```diff
@@ -1,2 +1,2 @@
-- compiler.warn.proc.messager: AnnotationFileParser: Unknown annotation: @Nullable
+- compiler.warn.proc.messager: AnnotationFileParser: Unknown annotation @Nullable in /home/mernst/research/types/checker-framework-fork-mernst-branch-AnnotationFileParser-diagnostics/checker/jtreg/stubs/issue2059/AnnoNotFound.astub
```

Comment:
@smillst Do you know offhand how to write this test so it will pass on all file systems?  (One way would be to add a flag, used only for debugging, that disables this output.)

Reply:
Maybe if `-Anomsgtext` is passed, then don't output the filename?